merge main to live

kexugit opened the pull request.

Commit:
CI Update

Commit:
Update generated content

Commit:
Update .openpublishing.publish.config.json

yunair committed:
update sdk

yunair committed:
test build

yunair committed:
wait master theme merge

yunair committed:
trigger build

yunair committed:
trigger build

yunair committed:
Merge pull request #7 from MicrosoftDocs/sdp

Replace legacy build pipeline with SDP build pipeline

v-pegao committed:
add `versions` for v3 migration

AngryBerryMS committed:
Merge pull request #8 from v-pegao/update-docfx-config

add `versions` for v3 migration

Commit:
Update config to enable docfx v3 (#9)

Update config to enable docfx v3

kenanlv committed:
add products taxonomy to improve findability

sudeepku committed:
Merge pull request #10 from kenanlv/master

Add products taxonomy to improve findability

herohua committed:
update with existing .openpublishing.publish.config.json

opbld30 commented:
Docs Build status updates of commit _[d65ca94](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/commits/d65ca948e0b4097a6ed6800fcd221ca58702254a)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-autogen/index.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/index.yml) | :x:Error | | [Details](#user-content-9c21406c1ba8af980f1b81b3dad27baf-8358f46c)
[docs-ref-autogen/BoundingBox.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/BoundingBox.yml) | :x:Error | | [Details](#user-content-e9836106984807e4f6c6d0d5f0f6b178-8358f46c)
[docs-ref-autogen/Map.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/Map.yml) | :x:Error | | [Details](#user-content-d46c6ed7708c51e24034c253c708fd3f-8358f46c)
[docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml) | :warning:Warning | | [Details](#user-content-ce30511dd95d5cbcdadf4240c8fa8633-8358f46c)
[docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml) | :warning:Warning | | [Details](#user-content-bc9ae08cd02409698ad897b3895dea3c-8358f46c)
[docs-ref-autogen/toc.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/toc.yml) | :warning:Warning | | [Details](#user-content-4082dcd77d2f312dffc12e9077c09669-8358f46c)
[docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml) | :bulb:Suggestion | | [Details](#user-content-6e8b5c1e3922a1939f0e83d81808a2c2-8358f46c)

<a id="9c21406c1ba8af980f1b81b3dad27baf-8358f46c"></a>
### [docs-ref-autogen/index.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/index.yml)
 - **Line 1, Column 1**: **[Error: schema-not-found]** `````Unknown schema 'UniversalReference'.`````

<a id="e9836106984807e4f6c6d0d5f0f6b178-8358f46c"></a>
### [docs-ref-autogen/BoundingBox.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/BoundingBox.yml)
 - **Line 1, Column 1**: **[Error: schema-not-found]** `````Unknown schema 'UniversalReference'.`````

<a id="d46c6ed7708c51e24034c253c708fd3f-8358f46c"></a>
### [docs-ref-autogen/Map.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/Map.yml)
 - **Line 1, Column 1**: **[Error: schema-not-found]** `````Unknown schema 'UniversalReference'.`````

<a id="ce30511dd95d5cbcdadf4240c8fa8633-8358f46c"></a>
### [docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml)
 - **Line 564, Column 54**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../operations/operations/get'.`````
 - **Line 585, Column 54**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../operations/operations/get'.`````

<a id="bc9ae08cd02409698ad897b3895dea3c-8358f46c"></a>
### [docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml)
 - **Line 954, Column 17**: **[Warning: xref-not-found]** `````Cross reference not found: 'Gallery.ExtensionStatsAggregateType'.`````

<a id="4082dcd77d2f312dffc12e9077c09669-8358f46c"></a>
### [docs-ref-autogen/toc.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/toc.yml)
 - **Line 1020, Column 16**: **[Warning: xref-not-found]** `````Cross reference not found: 'azure-devops-extension-api.CommentReportingList'.`````
 - **Line 2026, Column 16**: **[Warning: xref-not-found]** `````Cross reference not found: 'azure-devops-extension-api.FieldsToEvaluate'.`````
 - **Line 2599, Column 16**: **[Warning: xref-not-found]** `````Cross reference not found: 'azure-devops-extension-api.GraphMemberSearchFactor'.`````

<a id="6e8b5c1e3922a1939f0e83d81808a2c2-8358f46c"></a>
### [docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml)
 - **Line 26, Column 40**: **[Suggestion: disallowed-html-tag - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'publisher' isn't allowed.  Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````
 - **Line 26, Column 52**: **[Suggestion: disallowed-html-tag - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'extension' isn't allowed.  Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C11%5Cc511f339-8810-bf47-d1d9-fc58fc594b47%5CPullRequest%5C202203111628436438-12%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-11T16%3A29%3A57Z&sr=b&sp=r&sig=7y9WApMxbGilfrHP0dY0J6G%2BXUrCZgqv3P3njDkpjpI%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/c511f339-8810-bf47-d1d9-fc58fc594b47?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Commit:
Merge remote-tracking branch 'origin/live' into main

Commit:
merge live branch

opbld33 commented:
Docs Build status updates of commit _[e48284d](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/commits/e48284d7fc6c9b999685a82e6ee830a958853486)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml) | :warning:Warning | | [Details](#user-content-ce30511dd95d5cbcdadf4240c8fa8633-59d96015)
[docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml) | :warning:Warning | | [Details](#user-content-bc9ae08cd02409698ad897b3895dea3c-59d96015)
[docs-ref-autogen/toc.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/toc.yml) | :warning:Warning | | [Details](#user-content-4082dcd77d2f312dffc12e9077c09669-59d96015)
[docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml) | :bulb:Suggestion | | [Details](#user-content-6e8b5c1e3922a1939f0e83d81808a2c2-59d96015)
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[breadcrumb/toc.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/breadcrumb/toc.yml) | :white_check_mark:Succeeded | |
[docfx.json](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docfx.json) | :white_check_mark:Succeeded | |
[docs-ref-autogen/BoundingBox.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/BoundingBox.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/Map.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/Map.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AadLoginPromptOption.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AadLoginPromptOption.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AadOauthTokenRequest.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AadOauthTokenRequest.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AadOauthTokenResult.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AadOauthTokenResult.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AbortTestRunRequest.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AbortTestRunRequest.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccessMapping.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccessMapping.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccessTokenRequestType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccessTokenRequestType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/Account.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/Account.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountCreateInfoInternal.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountCreateInfoInternal.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountMyWorkResult.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountMyWorkResult.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountPreferencesInternal.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountPreferencesInternal.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel2.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel2.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModelBase.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModelBase.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentMentionWorkItemModel.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentMentionWorkItemModel.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountStatus.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountStatus.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountUserStatus.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountUserStatus.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountWorkWorkItemModel.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountWorkWorkItemModel.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountsRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountsRestClient.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionAssignmentType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionAssignmentType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperation.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperation.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationDisallowReason.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationDisallowReason.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationState.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationState.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOptions.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOptions.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/Activity.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/Activity.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/ActorFilter.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/ActorFilter.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/ActorNotificationReason.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/ActorNotificationReason.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AddProcessWorkItemTypeFieldRequest.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AddProcessWorkItemTypeFieldRequest.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AfnStrip.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AfnStrip.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentArtifactDefinition.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentArtifactDefinition.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentArtifactType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentArtifactType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentBasedDeployPhase.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentBasedDeployPhase.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentChangeEvent.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentChangeEvent.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentDeploymentInput.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentDeploymentInput.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentJobRequestMessage.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentJobRequestMessage.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentMigrationMessage.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentMigrationMessage.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolEvent.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolEvent.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolQueue.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolQueue.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueReference.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueReference.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueTarget.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueTarget.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
<a id="ce30511dd95d5cbcdadf4240c8fa8633-59d96015"></a>
### [docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml)
 - **Line 564, Column 54**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../operations/operations/get'.`````
 - **Line 585, Column 54**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../operations/operations/get'.`````

<a id="bc9ae08cd02409698ad897b3895dea3c-59d96015"></a>
### [docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml)
 - **Line 954, Column 17**: **[Warning: xref-not-found]** `````Cross reference not found: 'Gallery.ExtensionStatsAggregateType'.`````

<a id="4082dcd77d2f312dffc12e9077c09669-59d96015"></a>
### [docs-ref-autogen/toc.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/toc.yml)
 - **Line 1020, Column 16**: **[Warning: xref-not-found]** `````Cross reference not found: 'azure-devops-extension-api.CommentReportingList'.`````
 - **Line 2026, Column 16**: **[Warning: xref-not-found]** `````Cross reference not found: 'azure-devops-extension-api.FieldsToEvaluate'.`````
 - **Line 2599, Column 16**: **[Warning: xref-not-found]** `````Cross reference not found: 'azure-devops-extension-api.GraphMemberSearchFactor'.`````

<a id="6e8b5c1e3922a1939f0e83d81808a2c2-59d96015"></a>
### [docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml)
 - **Line 26, Column 40**: **[Suggestion: disallowed-html-tag - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'publisher' isn't allowed.  Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````
 - **Line 26, Column 52**: **[Suggestion: disallowed-html-tag - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'extension' isn't allowed.  Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C14%5Cc511f339-8810-bf47-d1d9-fc58fc594b47%5CPullRequest%5C202203140217341554-12%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-14T02%3A19%3A14Z&sr=b&sp=r&sig=Lnj%2B1DpnZrr3e2sJZEVH2qV1%2FNVBl%2FHd20%2B566%2B%2B2kM%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/c511f339-8810-bf47-d1d9-fc58fc594b47?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Commit:
CI Update
 Build.Reason:Manual by Yanan Zhao
 Build.Url:https://apidrop.visualstudio.com/Content%20CI/_build/results?buildId=283581&view=results

opbld31 commented:
Docs Build status updates of commit _[1d10936](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/commits/1d109363d0d95930b3ab6cc260f2d412bc81c38a)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml) | :warning:Warning | | [Details](#user-content-ce30511dd95d5cbcdadf4240c8fa8633-abf53f5c)
[docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml) | :warning:Warning | | [Details](#user-content-bc9ae08cd02409698ad897b3895dea3c-abf53f5c)
[docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml) | :bulb:Suggestion | | [Details](#user-content-6e8b5c1e3922a1939f0e83d81808a2c2-abf53f5c)
[.openpublishing.publish.config.json](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/.openpublishing.publish.config.json) | :white_check_mark:Succeeded | |
[breadcrumb/toc.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/breadcrumb/toc.yml) | :white_check_mark:Succeeded | |
[docfx.json](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docfx.json) | :white_check_mark:Succeeded | |
[docs-ref-autogen/BoundingBox.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/BoundingBox.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/Map.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/Map.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AadLoginPromptOption.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AadLoginPromptOption.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AadOauthTokenRequest.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AadOauthTokenRequest.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AadOauthTokenResult.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AadOauthTokenResult.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AbortTestRunRequest.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AbortTestRunRequest.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccessMapping.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccessMapping.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccessTokenRequestType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccessTokenRequestType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/Account.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/Account.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountCreateInfoInternal.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountCreateInfoInternal.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountMyWorkResult.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountMyWorkResult.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountPreferencesInternal.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountPreferencesInternal.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel2.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModel2.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModelBase.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentActivityWorkItemModelBase.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountRecentMentionWorkItemModel.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountRecentMentionWorkItemModel.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountStatus.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountStatus.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountUserStatus.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountUserStatus.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountWorkWorkItemModel.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountWorkWorkItemModel.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AccountsRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AccountsRestClient.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionAssignmentType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionAssignmentType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperation.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperation.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationDisallowReason.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationDisallowReason.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationState.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationState.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOperationType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AcquisitionOptions.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AcquisitionOptions.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/Activity.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/Activity.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/ActorFilter.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/ActorFilter.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/ActorNotificationReason.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/ActorNotificationReason.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AddProcessWorkItemTypeFieldRequest.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AddProcessWorkItemTypeFieldRequest.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AfnStrip.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AfnStrip.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentArtifactDefinition.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentArtifactDefinition.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentArtifactType.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentArtifactType.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentBasedDeployPhase.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentBasedDeployPhase.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentChangeEvent.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentChangeEvent.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentDeploymentInput.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentDeploymentInput.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentJobRequestMessage.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentJobRequestMessage.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentMigrationMessage.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentMigrationMessage.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolEvent.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolEvent.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolQueue.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolQueue.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueReference.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueReference.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueTarget.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentPoolQueueTarget.yml) | :white_check_mark:Succeeded | |
[docs-ref-autogen/azure-devops-extension-api/AgentQueueEvent.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/AgentQueueEvent.yml) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
<a id="ce30511dd95d5cbcdadf4240c8fa8633-abf53f5c"></a>
### [docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/CoreRestClient.yml)
 - **Line 609, Column 54**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../operations/operations/get'.`````
 - **Line 633, Column 54**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../../operations/operations/get'.`````

<a id="bc9ae08cd02409698ad897b3895dea3c-abf53f5c"></a>
### [docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-api/GalleryRestClient.yml)
 - **Line 1008, Column 17**: **[Warning: xref-not-found]** `````Cross reference not found: 'Gallery.ExtensionStatsAggregateType'.`````

<a id="6e8b5c1e3922a1939f0e83d81808a2c2-abf53f5c"></a>
### [docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml](https://github.com/MicrosoftDocs/azure-devops-docs-sdk-web/blob/main/docs-ref-autogen/azure-devops-extension-sdk/IExtensionContext.yml)
 - **Line 28, Column 40**: **[Suggestion: disallowed-html-tag - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'publisher' isn't allowed.  Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````
 - **Line 28, Column 52**: **[Suggestion: disallowed-html-tag - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/disallowed-html-tag?branch=main)]** `````HTML tag 'extension' isn't allowed.  Replace it with approved Markdown or escape the brackets if the content is a placeholder.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C21%5Cc511f339-8810-bf47-d1d9-fc58fc594b47%5CPullRequest%5C202203210149366974-12%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-21T01%3A51%3A03Z&sr=b&sp=r&sig=CXh2W12%2BZxxLVehoYZ7tuyvoYJdVDEc3CshWnP51yI0%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/c511f339-8810-bf47-d1d9-fc58fc594b47?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
